pahfit/base.py

```diff
@@ -512,14 +570,139 @@ def save(self, obs_fit, filename, outform):
+
         # stack the tables (handles missing columns between tables)
-        out_table = vstack([bb_table, line_table, att_table])
+        out_table = vstack([bb_table, line_table, att_table, cftable])
 
         # Writing output table
         out_table.write(
             "{}_output.{}".format(filename, outform), format=outform, overwrite=True
         )
 
+    @staticmethod
+    def featcombine(ftable):
+        """
+        Combine dust features strengths.
+
+        Parameters
+        ----------
+        ftable : data table
+            Astropy table containing lines.
+
+        Returns
+        -------
+        cftable : data table
+            Astropy table containing combined dust feature strengths.
+        """
+        # Define combined strength dictionary.
+        cfdic = {'PAH_62': {'range': [6.2, 6.3]},
+                 'PAH_77_C': {'range': [7.3, 7.9]},
+                 'PAH_83': {'range': [8.3, 8.4]},
+                 'PAH_86': {'range': [8.6, 8.7]},
+                 'PAH_112': {'range': [11.2, 11.4]},
+                 'PAH_120': {'range': [11.9, 12.1]},
+                 'PAH_126_C': {'range': [12.6, 12.7]},
+                 'PAH_136': {'range': [13.4, 13.6]},
+                 'PAH_142': {'range': [14.1, 14.2]},
+                 'PAH_164': {'range': [16.4, 16.5]},
+                 'PAH_17_C': {'range': [16.4, 17.9]},
+                 'PAH_174': {'range': [17.35, 17.45]}
+                 }
+
+        # Create combined strength, unc, and eqw table.
+        cftable = Table(
+            names=("Name", "x_0_min", "x_0_max", "strength", "strength_unc", "eqw"),
+            dtype=("U25", "float64", "float64", "float64", "float64", "float64"))
+
+        # Get indices of dust features.
+        df_ind = np.concatenate(np.argwhere(ftable["Form"] == "Drude1D"))
+
+        # Get table subset containing only the dust features.
+        dfs = ftable[df_ind[0]:df_ind[-1] + 1]
+
+        # Iterate keys and calculate combined strengths.
+        for feat in cfdic.keys():
+            mask = np.logical_and(dfs['x_0'] >= cfdic[feat]['range'][0], dfs['x_0'] <= cfdic[feat]['range'][-1])
+            cftable.add_row(
+                [feat,
+                 cfdic[feat]['range'][0],
+                 cfdic[feat]['range'][-1],
+                 np.sum(dfs[mask]['strength']),
+                 None,
+                 np.sum(dfs[mask]['eqw'])
+                 ])
+        return cftable
+
+    @staticmethod
+    def eqws(comp_type, x_0, amp, fwhm_stddev, obs_fit):
+        """
+        Calculate the emission features equivalent width in microns.
+
+        Parameters
+        ----------
+        comp_type : string
+            type of emission component (Drude1D/Gaussian)
+        x_0 : float
+            central wavelength of the feature.
+        amp : float
+            central intensity of the feature.
+        fwhm_stddev : float
+            fwhm or stddev of the feature depending on comp_type.
+
```

Comment:
Honestly I never trusted the EQW measured by main_feature_power.  I personally don't like EQW measures and don't use them (typically), but I also think this is squarely in the domain of "post-processing" that shouldn't be in the core fitting package.  A big picture way to think about it: we want to avoid making any scientific choices for end users, other than those expressed in our science packs.